website/content/en/status/report-2023-04-2023-06/mfsbsd.adoc

```diff
@@ -1,28 +1,28 @@
-It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems." 
+"mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, usb storage device or optical media.
+It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
 
-mailto:mm@FreeBSD.org[Martin Matuska] is both the author of the link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper] and the maintainer of the link:https://github.com/mmatuska/mfsbsd[mfsBSD repository]. 
+mailto:mm@FreeBSD.org[Martin Matuska] is both the author of the link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper] and the maintainer of the link:https://github.com/mmatuska/mfsbsd[mfsBSD repository].
 
-==== Purpose of this project 
+==== Purpose
 
-This project creates an additional target of the weekly snapshots of -current and -stable versions of mfsBSD images in the src/release makefile. 
+This project creates an additional target of the weekly snapshots of -current and -stable versions of mfsBSD images in the src/release makefile.
 Currently, only the release versions of mfsBSD images are produced, which means they tend to get out of sync with the tools in base. 
-This project aims to address that problem. 
+This project aims to address that problem.
 
-==== Location of this project 
+==== Location
 
-This project is a GSoC 2023 (Google Summer of Code) project, under mentorship from mailto:otis@FreeBSD.org[Juraj Lutter] and mailto:jrm@FreeBSD.org[Joseph Mingrone]. 
+This project is a GSoC 2023 (Google Summer of Code) project, under mentorship from mailto:otis@FreeBSD.org[Juraj Lutter] and mailto:jrm@FreeBSD.org[Joseph Mingrone].
 
-As a GSoC project, the official coding period is between May 29, 2023 and August 28, 2023. 
-As a humble beginner in the open-source community, the author of this project welcomes all comments / suggestions / PR's to the link:https://github.com/soobinrho/freebsd-src/tree/integrate-mfsBSD-building[project repository], which is where all the code for this project will be located for the duration of the official GSoC coding period. 
+As a GSoC project, the official coding period is between May 29, 2023 and August 28, 2023.
+As a humble beginner in the open-source community, the author of this project welcomes all comments / suggestions / pull requests to the link:https://github.com/soobinrho/freebsd-src/tree/integrate-mfsBSD-building[project repository], which is where all the code for this project will be located for the duration of the official GSoC coding period.
```

Comment:
Maybe move the project repository link to the **Links:** section.

Reply:
Resolved by [`7928b86` (#207)](https://github.com/freebsd/freebsd-doc/pull/207/commits/7928b8661e2387d0a28745e5ee73b9c0d4b05984).